lib/managers/enterprise.js

```diff
@@ -165,6 +165,35 @@ Enterprise.prototype.addUser = function(login, name, options, callback) {
 };
 
 /**
+ * Create a new app user in the current enterprise
+ *
+ * API Endpoint: '/users'
+ * Method: POST
+ *
+ * @param {string} name - The name of this user
+ * @param {?Object} options - Optional parameters, can be left null in most cases
+ * @param {string} [options.language] - The user's language
+ * @param {string} [options.job_title] - The user’s job title
+ * @param {string} [options.phone] - The user’s phone number
+ * @param {string} [options.address] - The user’s address
+ * @param {int} [options.space_amount] - The user’s total available storage space in bytes
+ * @param {string} [options.timezone] - The user's timezone
+ * @param {Function} callback - Passed the list of users if successful, error otherwise
+ * @returns {void}
+ */
+Enterprise.prototype.addAppUser = function(name, options, callback) {
+
+	var apiPath = urlPath(USERS_PATH),
+		params = {
+			body: {name, is_platform_access_only: true}
```

Comment:
We try to keep one object property per line, so it's easier to see what is being included and edit.  Would you mind splitting this onto two lines?